AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Haha funny referring to me. Make sure to mention that theres an exercise problem and its important they do it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I guess there won't be an exercise if we switch to Binomial here, and now I think we should switch</p><p> </p>

Reply:
<p>Great</p>